controllers/whoToFollow/whoToFollowV2.js

```diff
@@ -145,11 +145,11 @@ module.exports = async (req, res) => {
                                 a.is_anonymous = false
                                 AND a.is_banned = false
                                 AND a.blocked_by_admin = false
-                                AND a.karma_score > 30
+                                AND a.karma_score > 45
                                 AND b.location_id = :location_id
                               ORDER BY 
-                                a.last_active_at DESC,
-                                CASE WHEN a.profile_pic_path != '%default-profile-picture%' THEN 0 ELSE 1 END,
+                                CASE WHEN a.last_active_at > now() - interval '7' day THEN 0 ELSE 1 END,
+                                CASE WHEN a.profile_pic_path NOT LIKE '%default-profile-picture%' THEN 0 ELSE 1 END,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous hunk, the increase in the karma score threshold and the addition of the condition related to the last active time should help improve the relevance of the recommended users based on location. Again, be aware of the potential impact on the number of recommendations.

```diff
- AND a.karma_score > 30
+ AND a.karma_score > 45
- a.last_active_at DESC,
+ CASE WHEN a.last_active_at > now() - interval '7' day THEN 0 ELSE 1 END,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->